guacamole-docker/bin/start.sh

```diff
@@ -687,6 +687,18 @@ END
     ln -s /opt/guacamole/cas/guacamole-auth-*.jar   "$GUACAMOLE_EXT"
 }
 
+##
+## Adds properties to guacamole.properties which configure the json
+## authentication provider.
+##
+associate_json() {
+    # Update config file
+    set_optional_property "json-secret-key"         "$JSON_SECRET_KEY"
```

Comment:
Is this an optional property? Isn't this required if the JSON extension is used?

Reply:
Good catch. should be fixed now.